src/seedu/addressbook/data/person/Contact.java

```diff
@@ -0,0 +1,45 @@
+package seedu.addressbook.data.person;
+
+import seedu.addressbook.data.exception.IllegalValueException;
+
+/**
+ * Represents a Contact from which Person, Email and Address inherit 
```

Comment:
No need to mention subclasses in the header comment to the super class. You don't know who will inherit your super class in future and header comments become outdated if someone uses it for another subclass.